assets/scss/templates/_code.scss

```diff
@@ -0,0 +1,31 @@
+div.highlight {
+  margin: 1.5rem 0;
+  background: #f7f7f8;
+  overflow-x: auto;
+  position: relative;
+}
+
+code {
+  color: #444;
+  margin: 0;
+}
+
+pre {
+  padding: 1rem;
+  margin: 0;
+  border: 0;
+  border-radius: 2px;
+  line-height: 1.15;
+
+
+  code {
+    display: block;
+    color: #333;
+    background: inherit;
+    white-space: inherit;
+    border: 0;
+    padding: 0.5em; // same as in highlighter
```

Comment:
Do we really want the padding to adapt depending on font-size? Feel free to resolve this comment if yes

Reply:
This is probably related to code blocks within a callout or notice, where the font size would be smaller